### DISCUSSION TITLE
ID de los pinguinos

### DISCUSSION THREAD

**Discussion by IgnacioHqz** [IIC2133-PUC/2024-2#124] [Tarea 1] [answered]
Hola, quisiera saber si los id de los pinguinos siempre sera 0, 1, 2, 3, y así sucesivamente para cada ENTER que se agregue? Es decir, si en el sector 0 ingresan 3 pinguinos con id 0, 1 y 2, esos id no se repetiran en ningún otro sector o es posible que por ejemplo en el sector 3 los pinguinos que ingresen partan con un id igual a 0? Ojalá me pudieran orientar. Gracias

**Comment by sebacarrascop** [comment, marked as answer]
Hola si puedes asumir eso, eso igual se puede apreciar en los tests que están subidos en el drive,
Saludos.
